db.json-schema.json

```diff
@@ -214,6 +214,10 @@
                     "generalLocationName": {
                       "type": "string",
                       "description": "A name for the location, generally 'City, State'."
+                    },
+                    "hasAccount": {
```

Comment:
Can we change this to something like `profileFilledOut`?

Reply:
Sure